provider/cmd/pulumi-resource-nuage/nuage_provider/container_function.py

```diff
@@ -168,12 +202,47 @@ def __init__(self, name: str, args: ContainerFunctionArgs, props: Optional[dict]
             )
         )
         
+        # FIXME: Thhis doesn't work because pulumi Command doesn't have option to run on update right now 
+        # As it work only on create, it keeps having the same tag on updates
+        #untag_command = local.Command(
+        #    f"untag-{name}-image",
+        #    create=repository_url.apply(
+        #        lambda repository_url: f"docker rmi {repository_url}"
+        #    ), 
+        #    opts=pulumi.ResourceOptions(depends_on=[image])
+        #)
+        
         # Define inline policies for role definition
+        log_group = aws.cloudwatch.LogGroup(
+            f"{name}-log-group", 
+            name=f"/aws/lambda/{name}",
+            retention_in_days=90
+        )
+        lambda_logging = aws.iam.Policy(
+            f"{name}-logging-policy",
+            path="/",
+            description="IAM policy for logging from a lambda",
+            policy="""{
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                "Action": [
+                    "logs:CreateLogStream",
+                    "logs:PutLogEvents"
+                ],
+                "Resource": "arn:aws:logs:*:*:*",
+                "Effect": "Allow"
+                }
+            ]
+            }
+            """
+        )
+        
         policy_documents = [
             # Can write logs to CloudWatch
             aws.iam.RoleInlinePolicyArgs(   
-                name="PolicyAWSLambdaBasicExecutionRole",
-                policy=aws.iam.get_policy(name="AWSLambdaBasicExecutionRole").policy,
+                name=f"{name}-logging-policy",
+                policy=lambda_logging.policy,
```

Comment:
Maybe it'd be better to inline the policy using `get_policy_document().json` no ?

It'd create one less resource (I think)